code/modules/food&drinks/drinks/drinks.dm

```diff
@@ -288,3 +298,302 @@
 	desc = "A delicious mixture of 42 different flavors."
 	icon_state = "dr_gibb"
 	list_reagents = list("dr_gibb" = 30)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/pubber
+	name = "Dr. Pubber"
+	desc = "The beverage of an original crowd. Tastes like an industrial tranquilizer."
+	icon_state = "dr_gibb"
+	New()
+		..()
+		reagents.add_reagent("haloperidol", 4)
+		reagents.add_reagent("morphine", 4)
+		reagents.add_reagent("vhfcs", 10)
+		reagents.add_reagent("cola", 12)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/robust
+	name = "Robust-Eeze"
+	desc = "A carbonated robustness tonic. It has quite a kick."
+	icon_state = "cola"
+	New()
+		..()
+		reagents.add_reagent("methamphetamine", 3)
+		reagents.add_reagent("vhfcs", 10)
+		reagents.add_reagent("cola", 17)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/grifeo
+	name = "Grife-O"
+	desc = "The carbonated beverage of a space generation. Contains actual space dust!"
+	icon_state = "griefo"
+	New()
+		..()
+		reagents.add_reagent("radium", 3)
+		reagents.add_reagent("ephedrine", 6)
+		reagents.add_reagent("vhfcs", 10)
+		reagents.add_reagent("cola", 11)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/orangeaid
+	name = "Orange-Aid"
+	desc = "A vitamin tonic that promotes good eyesight and health."
+	icon_state = "starkist"
+	New()
+		..()
+		reagents.add_reagent("orangejuice", 20)
+		reagents.add_reagent("oculine", 20)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/limeaid
+	name = "Lime-Aid"
+	desc = "Antihol mixed with lime juice. A well-known cure for hangovers."
+	icon_state = "space-up"
+	New()
+		..()
+		reagents.add_reagent("limejuice", 20)
+		reagents.add_reagent("antihol", 20)
+
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/runoff
+	name = "Spooky Dan's Runoffa Cola"
+	desc = "A spoooky cola for Halloween!  Rumors that Runoff Cola contains actual industrial runoff are unsubstantiated."
+	icon_state = "purple_can"
+	New()
+		..()
+		reagents.add_reagent("chlorine", 5)
+		reagents.add_reagent("phosphorus", 5)
+		reagents.add_reagent("mercury", 5)
```

Comment:
This is a fucking drink you moron. A drink. For people to consume.

"Issue 9987: Character died for no reason "
"sorry man, goof put poisons in random drinks"